bodhi/server/models.py

```diff
@@ -791,7 +791,7 @@ class Build(Base):
     """
     __tablename__ = 'builds'
     __exclude_columns__ = ('id', 'package', 'package_id', 'release',
-                           'release_id', 'update_id', 'update', 'override')
+                           'update_id', 'update', 'override')
```

Comment:
I am a bit on the fence about doing this. The downside is that it is exposing what might be considered internal primary keys to API users. The alternative would be to either pass the release along with the ```Build``` object (which would result in a lot more data when users query for Builds) or to add a ```release_name``` property (with the ```release.name```, the client could use the existing ```releases/{{ name }}``` API to get the needed information). Both of these alternatives would cause a join to happen any time Builds are serialized.

In this pull request I opted to give the ```releases/``` API the ability to query by primary key to avoid the join every time Builds are serialized.